packages/web3torrent/src/App.tsx

```diff
@@ -55,6 +83,16 @@ const App: React.FC = () => {
           requiredNetwork={requiredNetwork}
           onWeb3Fallback={!('ethereum' in window)}
         />
+        {ready && balance && balance.lt(INITIAL_BUDGET_AMOUNT) && (
+          <Flash my={3} variant="warning">
+            You currently have {utils.formatEther(balance)} ETH in your wallet. You'll need at least{' '}
+            {utils.formatEther(INITIAL_BUDGET_AMOUNT)} ETH in your wallet to fund Web3Torrent. You
```

Comment:
Not sure I am a fan of how many 0s this outputs for small amounts

Reply:
I was thinking about that too. However two reasons I stuck with it.
- The approve and fund workflow displays the same thing when showing the budget.
- I think more people know `ETH than `Mwei` or other denominations of eth.